content/influxdb/v2.0/security/tokens/_index.md

```diff
@@ -12,7 +12,7 @@ menu:
 weight: 103
 ---
 
-InfluxDB ensures secure interaction between users and data through the use of **authentication tokens**.
+InfluxDB **authentication tokens** ensure secure interaction between users and data.
```

Comment:
can we start calling these API tokens?

Reply:
This is a widespread change, so I'm going to reserve this for another PR.